packages/gatsby-background-image/README.md

```diff
@@ -641,6 +641,48 @@ props from `gatsby-image` are not available, of course.
 | `placeholderClassName` | `string` | A class that is passed to the placeholder img element         |
 | `imgStyle`             | `object` | Spread into the default styles of the actual img element      |
 
+In the absence of the `placeholderStyle` prop, additional styling while the image is loading can be accomplished using the `onLoad` or `onStartLoad` props. Use either method's callback to toggle a className on the component with your loading styles.
+
+An example of "softening" the blur up using vanilla CSS.
+
+```
+/* MyBackgroundImage.css */
+
+.loading,
+.loading::before,
+.loading::after {
+  filter: blur(15px);
+}
+
+/* ...other styles */
+```
+
+```
+// MyBackroundImage.js
+
+import React, { useRef } from "react"
+import BackgroundImage from "gatsby-background-image-es5"
```

Comment:
Only change necessary: remove -es5 in `gbi`s README; ).

Reply:
🤦🏻‍♂️fixed. sorry, that was careless.